libs/types/src/election.ts

```diff
@@ -434,9 +413,9 @@ function copyWithLocale<T>(
   locale: string
 ): T | readonly T[] {
   if (Array.isArray(value)) {
-    return (value.map((element) =>
-      copyWithLocale(element, locale)
-    ) as unknown) as T
+    return value.map(
+      (element) => (copyWithLocale(element, locale) as unknown) as T
```

Comment:
This was just wrong before, as I had it casting the _array_ as `unknown` then `T`, vs this casts the _elements_.